<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="XYImageShower" name="show">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(229, 229, 229);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>放大</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>缩小</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>显示边框</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QLineEdit" name="width">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>高：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>选择文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>加载</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="imagepath"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>图片格式：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>宽：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Format_Mono</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_MonoLSB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_Indexed8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_RGB32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_ARGB32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_ARGB32_Premultiplied</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_RGB16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_ARGB8565_Premultiplied</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_RGB666</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_ARGB6666_Premultiplied</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_RGB555</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_ARGB8555_Premultiplied</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_RGB888</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_RGB444</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_ARGB4444_Premultiplied</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_RGBX8888</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_RGBA8888</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_RGBA8888_Premultiplied</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_BGR30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_A2BGR30_Premultiplied</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_RGB30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_A2RGB30_Premultiplied</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_Alpha8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format_Grayscale8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>图片路径：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QTextEdit" name="textEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="XYImageContainer" name="contain" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="height">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XYImageShower</class>
   <extends>QFrame</extends>
   <header>xyimageshower.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XYImageContainer</class>
   <extends>QWidget</extends>
   <header>xyimagecontainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
